.github/workflows/make_release.yml

```
@@ -44,15 +44,33 @@ jobs:
           fi
           echo "tag=${TAG}" >> $GITHUB_ENV
           # https://help.github.com/en/actions/reference/workflow-commands-for-github-actions
-          echo "name=contents=${RELEASE_NOTES}" >> $GITHUB_ENV
+          echo "contents=${RELEASE_NOTES}" >> "$GITHUB_OUTPUT"
+
+      - name: check if prerelease
+        id: prerelease
+        run: |
+          regex='^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+([ab][0-9]+|rc[0-9]+)$'
+
+          check_version() {
+              if [[ $1 =~ $regex ]]; then
+                  echo "true"
+              else
+                  echo "false"
+              fi
+          }
+          echo ${{ github.ref }}
+          check_version ${{ github.ref }}
+          echo "prerelease=$(check_version ${{ github.ref }})" >> "$GITHUB_OUTPUT"
+        shell: bash
+
       - name: Create Release
         uses: "softprops/action-gh-release@v2"
         with:
           tag_name: ${{ github.ref }}
           name: ${{ env.tag }}
           body: ${{ steps.release_notes.outputs.contents }}
           draft: false
-          prerelease: ${{ contains(github.ref, 'rc') }}
+          prerelease: ${{ steps.prerelease.outputs.prerelease  == 'true' }}
```

I'm ok with this change but it seems a bit more complex than necessary... Why not this change alone?

```suggestion
          prerelease: ${{ contains(github.ref, 'rc') || contains(github.ref, 'a') || contains(github.ref, 'b') }}
```

---

See how `github.ref` looks like a valid release `refs/tags/v0.5.0` and why checking if `a` is in a string is not a good idea.

---

LOL Definitely missed that, I thought it was just `v0.5.0`. 😅

---

It will be much simpler if we could check only tag. But I do not found a way to do this.